openmc/settings.py

```diff
@@ -1087,3 +1118,101 @@ def export_to_xml(self):
         tree = ET.ElementTree(self._settings_file)
         tree.write("settings.xml", xml_declaration=True,
                              encoding='utf-8', method="xml")
+
+
+class ResonanceScattering(object):
+    """Specification of the elastic scattering model for resonant isotopes.
+
+    Attributes
+    ----------
+    nuclide : openmc.nuclide.Nuclide
```

Comment:
Ditto, remove periods after each attribute's comment (unless it is a multi-sentence comment) for consistency.